diff -ruN tome100-src.orig/src/init2.c tome100-src/src/init2.c
--- tome100-src.orig/src/init2.c	Sun May  5 00:45:44 2002
+++ tome100-src/src/init2.c	Tue May 14 17:21:51 2002
@@ -72,7 +72,10 @@
 void init_file_paths(char *path)
 {
 	char *tail;
+	char *tail2;
+	char path2[256];
 
+	strcpy(path2, VAR_PATH);
 
 	/*** Free everything ***/
 
@@ -104,6 +107,7 @@
 
 	/* Prepare to append to the Base Path */
 	tail = path + strlen(path);
+	tail2 = path2 + strlen(path2);
 
 
 #ifdef VM
@@ -133,20 +137,20 @@
 	/*** Build the sub-directory names ***/
 
 	/* Build a path name */
-	strcpy(tail, "apex");
-	ANGBAND_DIR_APEX = string_make(path);
+	strcpy(tail2, "apex");
+	ANGBAND_DIR_APEX = string_make(path2);
 
 	/* Build a path name */
-	strcpy(tail, "bone");
-	ANGBAND_DIR_BONE = string_make(path);
+	strcpy(tail2, "bone");
+	ANGBAND_DIR_BONE = string_make(path2);
 
 	/* Build a path name */
         strcpy(tail, "dngn");
         ANGBAND_DIR_DNGN = string_make(path);
 
 	/* Build a path name */
-	strcpy(tail, "data");
-	ANGBAND_DIR_DATA = string_make(path);
+	strcpy(tail2, "data");
+	ANGBAND_DIR_DATA = string_make(path2);
 
 	/* Build a path name */
 	strcpy(tail, "edit");
@@ -169,8 +173,8 @@
         ANGBAND_DIR_NOTE = string_make(path);
 
 	/* Build a path name */
-	strcpy(tail, "save");
-	ANGBAND_DIR_SAVE = string_make(path);
+	strcpy(tail2, "save");
+	ANGBAND_DIR_SAVE = string_make(path2);
 
 	/* Build a path name */
         strcpy(tail, "scpt");
diff -ruN tome100-src.orig/src/makefile.org tome100-src/src/makefile.org
--- tome100-src.orig/src/makefile.org	Tue May 14 17:11:37 2002
+++ tome100-src/src/makefile.org	Tue May 14 17:11:51 2002
@@ -75,6 +75,7 @@
 
 # Where lib/ files goes
 LIBDIR = /usr/share/games/tome/
+VARDIR = /var/games/tome/
 # Sysadmins of commercial Unix and/or BSD might prefer this
 #LIBDIR = /usr/local/lib/tome/
 # If you like the old default, use this one
@@ -345,7 +346,7 @@
 # Compiler options
 #
 
-CFLAGS = $(COPTS) $(INCLUDES) $(DEFINES) -DDEFAULT_PATH=\"$(LIBDIR)\" $(IRCARGS)
+CFLAGS = $(COPTS) $(INCLUDES) $(DEFINES) -DVAR_PATH=\"$(VARDIR)\" -DDEFAULT_PATH=\"$(LIBDIR)\" $(IRCARGS)
 
 
 #
