--- tome/src/makefile.std~	2004-01-16 22:17:12.951756240 +0100
+++ tome/src/makefile.std	2004-01-16 22:33:00.685308461 +0100
@@ -78,6 +78,8 @@
 
 # Where lib/ files goes
 LIBDIR = /usr/share/games/tome/
+VARDIR = /var/games/tome/
+
 # Sysadmins of commercial Unix and/or BSD might prefer this
 #LIBDIR = /usr/local/lib/tome/
 # If you like the old default, use this one
@@ -383,7 +385,7 @@
 # Compiler options
 #
 
-CFLAGS = $(COPTS) $(PROFILECOPTS) $(INCLUDES) $(DEFINES) -DDEFAULT_PATH=\"$(LIBDIR)\"
+CFLAGS = $(COPTS) $(PROFILECOPTS) $(INCLUDES) $(DEFINES) -DDEFAULT_PATH=\"$(LIBDIR)\" -DVAR_PATH=\"$(VARDIR)\"
 
 
 #
--- tome/src/init2.c~	2003-12-11 21:32:41.000000000 +0100
+++ tome/src/init2.c	2004-01-16 22:32:06.689380960 +0100
@@ -71,8 +71,10 @@
  */
 void init_file_paths(char *path)
 {
-	char *tail;
-	int   pathlen;
+	char *tail,*tail2;
+	int   pathlen,pathlen2;
+	char path2[256];
+	strcpy(path2, VAR_PATH);
 
 	/*** Free everything ***/
 
@@ -103,6 +105,7 @@
 	/*** Prepare the "path" ***/
 
 	pathlen = strlen(path);
+	pathlen2 = strlen(path2);
 
 	/* Hack -- save the main directory without trailing PATH_SEP if present */
 	if (strlen(PATH_SEP) > 0 && pathlen > 0)
@@ -127,6 +130,7 @@
 
 	/* Prepare to append to the Base Path */
 	tail = path + pathlen;
+	tail2 = path2 + pathlen2;
 
 
 
@@ -160,12 +164,12 @@
 	/*** Build the sub-directory names ***/
 
 	/* Build a path name */
-	strcpy(tail, "apex");
-	ANGBAND_DIR_APEX = string_make(path);
+	strcpy(tail2, "apex");
+	ANGBAND_DIR_APEX = string_make(path2);
 
 	/* Build a path name */
-	strcpy(tail, "bone");
-	ANGBAND_DIR_BONE = string_make(path);
+	strcpy(tail2, "bone");
+	ANGBAND_DIR_BONE = string_make(path2);
 
 	/* Build a path name */
 	strcpy(tail, "core");
@@ -176,8 +180,8 @@
 	ANGBAND_DIR_DNGN = string_make(path);
 
 	/* Build a path name */
-	strcpy(tail, "data");
-	ANGBAND_DIR_DATA = string_make(path);
+	strcpy(tail2, "data");
+	ANGBAND_DIR_DATA = string_make(path2);
 
 	/* Build a path name */
 	strcpy(tail, "edit");
@@ -232,8 +236,8 @@
 #else /* PRIVATE_USER_PATH */
 
 	/* Build a path name */
-	strcpy(tail, "save");
-	ANGBAND_DIR_SAVE = string_make(path);
+	strcpy(tail2, "save");
+	ANGBAND_DIR_SAVE = string_make(path2);
 
 	/* Build a path name */
 	strcpy(tail, "user");
