--- tome-300alpha17-src/src/makefile.std~	2007-10-25 22:43:34.000000000 +0200
+++ tome-300alpha17-src/src/makefile.std	2007-10-25 22:43:14.000000000 +0200
@@ -30,9 +30,9 @@
 # tome.cfg needs editing to point to the correct location
 
 # Where lib/ and tome.cfg files goes
-LIBDIR = $(PREFIX)/games/$(TOMENAME)/
+#LIBDIR = $(PREFIX)/games/$(TOMENAME)/
 # Sysadmins of commercial Unix and/or BSD might prefer this
-#LIBDIR = $(PREFIX)/share/$(TOMENAME)/
+LIBDIR = $(PREFIX)/share/games/$(TOMENAME)/
 # Another example: single user installation using absolute path
 #LIBDIR = /home/myloginname/lib/$(TOMENAME)/
 # If none are defined then tome will look for tome.cfg in the same
@@ -106,17 +106,6 @@
 # including "USE_GETCH" and "USE_CURS_SET".  Note that "config.h" will
 # attempt to "guess" at many of these flags based on your system.
 #
-COPTS = -Wall -O1 -pipe -g
-INCLUDES = -I$(X11BASE)/include
-DEFINES = -DUSE_X11 \
-	-DUSE_GCU \
-	-DUSE_EGO_GRAPHICS \
-	-DUSE_TRANSPARENCY \
-	-DSUPPORT_GAMMA \
-	-DUSE_PRECISE_CMOVIE \
-	-DUSE_UNIXSOCK
-LIBS = -lcurses -L$(X11BASE)/lib -lX11
-
 
 ##
 ## Variation -- Use "main-sdl.c" instead of "main-x11.c"
@@ -153,10 +142,10 @@
 ##
 ## Variation -- Only support "main-gcu.c" (not "main-x11.c")
 ##
-#COPTS = -Wall -O1 -pipe -g
-#INCLUDES =
-#DEFINES = -DUSE_GCU
-#LIBS = -lcurses -ltermcap
+COPTS = -Wall -O1 -pipe -g
+INCLUDES = -I/usr/include/ncurses
+DEFINES = -DUSE_GCU -DUSE_NCURSES
+LIBS = -lcurses
 
 
 ##
