--- tome100-src-org/src/makefile.org	Sun May  5 00:45:44 2002
+++ tome100-src/src/makefile.org	Thu May  9 00:21:02 2002
@@ -56,7 +56,7 @@
 # Comment out this line if you don't want Lua scripting
 LUA=TRUE
 
-IRCARGS = -DUSE_SOCK -DUSE_UNIXSOCK -DIRC_SERVER=\"irc.worldirc.org\" -DIRC_PORT=\"6667\" -DIRC_CHANNEL=\"\#*bandirc\"
+#IRCARGS = -DUSE_SOCK -DUSE_UNIXSOCK -DIRC_SERVER=\"irc.worldirc.org\" -DIRC_PORT=\"6667\" -DIRC_CHANNEL=\"\#*bandirc\"
 
 
 ##
@@ -74,7 +74,7 @@
 ##
 
 # Where lib/ files goes
-LIBDIR = /usr/lib/games/tome/
+LIBDIR = /usr/share/games/tome/
 # Sysadmins of commercial Unix and/or BSD might prefer this
 #LIBDIR = /usr/local/lib/tome/
 # If you like the old default, use this one
@@ -84,7 +84,7 @@
 
 
 # Where T.o.M.E. binary goes
-BINDIR = /usr/local/games
+BINDIR = /usr/games
 # Another common location
 #BINDIR = /usr/local/bin
 
@@ -142,10 +142,10 @@
 # including "USE_GETCH" and "USE_CURS_SET".  Note that "config.h" will
 # attempt to "guess" at many of these flags based on your system.
 #
-COPTS = -Wall -O1 -pipe -g
-INCLUDES = -I/usr/X11R6/include
-DEFINES = -DUSE_X11 -DUSE_GCU -DSUPPORT_GAMMA
-LIBS = -lX11 -lcurses -L/usr/X11R6/lib
+#COPTS = -Wall -O1 -pipe -g
+#INCLUDES = -I/usr/X11R6/include
+#DEFINES = -DUSE_NCURSES -DUSE_X11 -DUSE_GCU -DSUPPORT_GAMMA
+#LIBS = -lX11 -lncurses -L/usr/X11R6/lib
 
 
 ##
@@ -168,10 +168,10 @@
 ##
 ## Variation -- Only support "main-gcu.c" (not "main-x11.c")
 ##
-#COPTS = -Wall -O1 -pipe -g
-#INCLUDES =
-#DEFINES = -DUSE_GCU
-#LIBS = -lcurses -ltermcap
+COPTS = -Wall -O1 -pipe -g
+INCLUDES = -I/usr/include/ncurses
+DEFINES = -DUSE_GCU -DUSE_NCURSES
+LIBS = -lncurses
 
 
 ##
@@ -393,13 +393,13 @@
 	cp -f $(TARGET) ..
 
 install: default
-	[ -d $(LIBDIR) ] || mkdir -p $(LIBDIR)
-	[ -d $(BINDIR) ] || mkdir -p $(BINDIR)
-	cp -r ../lib/* $(LIBDIR)
-	chown -R $(OWNER) $(LIBDIR)
-	cp -f $(TARGET) $(BINDIR)/$(TARGET)
-	chown $(OWNER) $(BINDIR)/$(TARGET)
-	chmod 4755 $(BINDIR)/$(TARGET)
+	[ -d $(DESTDIR)$(LIBDIR) ] || mkdir -p $(DESTDIR)$(LIBDIR)
+	[ -d $(DESTDIR)$(BINDIR) ] || mkdir -p $(DESTDIR)$(BINDIR)
+	cp -r ../lib/* $(DESTDIR)$(LIBDIR)
+#	chown -R $(OWNER) $(DESTDIR)$(LIBDIR)
+	cp -f $(TARGET) $(DESTDIR)$(BINDIR)/$(TARGET)
+#	chown $(OWNER) $(DESTDIR)$(BINDIR)/$(TARGET)
+#	chmod 4755 $(DESTDIR)$(BINDIR)/$(TARGET)
 
 # old-install: $(TARGET)
 #	cp $(TARGET) ..
